app/controllers/users_controller.rb

```diff
@@ -257,6 +257,7 @@ def handle_organization_tab
     @organizations = @current_user.organizations.order(name: :asc)
     if params[:org_id] == "new" || (params[:org_id].blank? && @organizations.empty?)
       @organization = Organization.new
+      @is_new_organization = true
```

Comment:
You can also just check if it's a new or existing organization using `@organization&.new_record?`